code/__DEFINES/colours.dm

```diff
@@ -33,40 +33,49 @@
 /// full cyan rgb(0, 255, 255) B + G
 #define COLOR_CYAN "#00FFFF"
 /// full magenta(not pink) rgb(255, 0, 255) R+B
-#define COLOR_PINK "#FF00FF"
+#define COLOR_MAGENTA "#FF00FF"
 /// full yellow rgb(255, 255, 0) R+G
 #define COLOR_YELLOW "#FFFF00"
 
 // colors define in use bellow
+
+
+//color define with a proper name
+
+///Olive rgb(128, 128, 0)
+#define COLOR_OLIVE "#808000"
+///Silver rgb(192, 192, 192) shade of grey
 #define COLOR_SILVER "#C0C0C0"
+///Gray rgb(128, 128, 128)
+#define COLOR_GRAY "#808080"
 
 #define COLOR_FLOORTILE_GRAY "#8D8B8B"
 
 #define COLOR_HALF_TRANSPARENT_BLACK "#0000007A"
 
 #define COLOR_DARK_RED "#A50824"
 
+///Maroon rgb(128, 0, 0) shade of red
 #define COLOR_MAROON "#800000"
+
 #define COLOR_VIVID_RED "#FF3232"
 #define COLOR_LIGHT_GRAYISH_RED "#E4C7C5"
 #define COLOR_SOFT_RED "#FA8282"
 
 #define COLOR_VERY_SOFT_YELLOW "#FAE48E"
 
-#define COLOR_OLIVE "#808000"
-
 ///light green rgb( 0, 128, 0)
 #define COLOR_LIGHT_GREEN "#008000"
 #define COLOR_DARK_MODERATE_LIME_GREEN "#44964A"
 
 #define COLOR_TEAL "#008080"
 
 #define COLOR_MODERATE_BLUE "#555CC2"
-
+///Purple rgb( 128, 0, 128)
 #define COLOR_PURPLE "#800080"
 #define COLOR_STRONG_VIOLET "#6927c5"
 
-#define COLOR_BEIGE "#CEB689"
+#define COLOR_LIGHT_ORANGE "#CEB689"
```

Comment:
Not sure I would call this light orange myself

https://www.color-hex.com/color-palette/9777

Reply:
because it's a shade of orange on the site i used for color https://webdi.fr/couleur-hexa.php?c=%23CEB689
and nothing was called light orange so....
if you have a better name for it because calling it beige was inaccurate as beige is this https://webdi.fr/couleur-hexa.php?c=c8ad7f 
#C8AD7F
first time i take a look in colors and all... so i don't know maybe LIGHT_BEIGE?

Reply:
would LIGHT_BEIGE be fine instead don't want to called it beige when it's not the color...

Reply:
done